docs/modules/ROOT/pages/clc-serializer-generator.adoc

```diff
@@ -0,0 +1,136 @@
+= clc serializer generator (Beta)
+
+Serializer commands are a group of serializer generator operations.
+
+Check out the documentation for https://docs.hazelcast.com/hazelcast/latest/serialization/compact-serialization[Compact Serialization] for more information.
+
+Usage:
+
+[source,bash]
+----
+clc serializer [command] [flags]
+----
+
+== Commands
+
+* <<clc-serializer-generate, clc serializer generate>>
+
+== clc serializer generate
+
+Generates compact serialization code from the given schema for the target language. See https://docs.hazelcast.com/hazelcast/latest/serialization/compact-serialization#implementing-compactserializer[Compact Serialization] documentation for more information.
+
+Usage:
+
+[source, bash]
+----
+clc serializer generate [schema] [flags]
+----
+
+Parameters:
+
+[cols="1m,1a,2a,1a"]
+|===
+|Parameter|Required|Description|Default
+
+|`output-dir`, `-o`
+|Optional
+|Output directory for the serialization files.
+|Current working directory.
+
+|`language`, `-l`
+|Required
+|Programming language that the serializer created for.
+|
+|===
+
+Example:
+
+[source,bash]
+----
+clc serializer generate my-schema.yaml --language java --output-dir my-dir
+----
+
+=== Schema Creation
+
+A schema allows you to:
+
+* describe the contents of a compact class using supported field types
+* import other schema
+* specify a namespaces for schema files and reference other namespaces
+* define cyclic references between classes
+* reference classes that are not present in the given schemas
+
+A schema is written in YAML. Schema format is given below:
+
+[source,yaml]
+----
+namespace: <namespace of the class>
+# note that other schema files can be imported with relative path to this yaml file
+imports:
+ - someOtherSchema.yaml
+# All objects in this file will share the same namespace.
+classes:
+ - name: <name of the class>
+   fields:
+     - name: <field name>
+       type: <field type>
+       external: bool # to mark external types (external to this yaml file)
+----
+
+==== namespace:
```

Comment:
Could you add a blank line after headers and remove the oclumns?

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/406/commits/29ec891eadaca81a0a738584a071ccf8072acab7